### ISSUE TITLE
Grease Pencil Animations?

### ISSUE THREAD

**Issue by 3DEsprit** [Fweeb/blender_spritify#8]
Hey man, I know this wasn't made for GP, but I thought I would at least bring it up. Any way to have an option to search for GP animations, since GPs are now their own objects? This could make for some great 2D animation implementations. ;)

**Comment by Fweeb** [comment]
I *think* I understand what you're asking for, but a bit more clarification would be appreciated.

AFAIK, in order to render Grease Pencil animations, you basically have to do an OpenGL render. Currently Spritify only works with the primary renderers (Cycles or BI). Technically, getting it to work with OpenGL rendering shouldn't be hard, but since you can't choose OpenGL as a renderer like you can Cycles or BI, I'm not sure what the best way to handle the UI for this would be. A separate Grease Pencil toggle/button feels kludgy.

**Comment by 3DEsprit** [comment]
Yep, you get it. Ah, I completely forgot that it couldn't be rendered. 

Of course you always have the OpenGL render buttons in the 3D view Render Menu, so no need to add another button. ;)
